specs/protocol/isthmus/exec-engine.md

```diff
@@ -208,17 +212,57 @@ and the `operatorFeeConstant`.
 
 ### Operator Fee
 
-The operator fee, is set as follows:
+The operator fee is integrated directly into the EVM, alongside the standard gas fee and the OP Stack specific L1 data
+fee. This fee must follow the exact same semantics as existing fees charged in the EVM.
```

Comment:
> This fee must follow the exact same semantics as existing fees charged in the EVM.

Not sure what this means. The operator fee is not applied in the exact same way tx costs are applied - fees are accrued to a new operator fee recipient while the charge and refund apply to the tx sender. Maybe there's something we could point to here to make this more clear, otherwise it should be deleted imo as it doesn't add anything concrete.

Reply:
I actually prefer the route you take comparing the fee pattern.

Maybe it's enough just to remove "exact same". Might I suggest the following.

```suggestion
This follows the semantics of existing fees charged in the EVM.
```

Reply:
Perf - will update!